packages/office-ui-fabric-react/src/components/Dialog/DialogContent.types.ts

```diff
@@ -77,6 +79,11 @@ export interface IDialogContentProps extends React.ClassAttributes<DialogContent
    */
   title?: string | JSX.Element;
 
+  /**
+   * The props for title container.
+   */
+  titleProps?: React.AllHTMLAttributes<HTMLDivElement>;
```

Comment:
Sorry for not noticing this earlier but I think this should just be `React.HTMLAttributes` for correctness and consistency. I'll approve for now

Reply:
ah good catch. my bad. will fix